blueprints/serverless/cloud-run-microservices/cloudrun.tf

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# tfdoc:file:description Cloud Run services.
+
+# Cloud Run service A
+resource "google_cloud_run_v2_service" "svc_a" {
```

Comment:
What is the rationale here to use raw resources instead of `modules/cloud-run`?

Reply:
Beta features (Direct VPC Egress), the cloud run module uses resource "google_cloud_run_service" that doesn't support it.

Reply:
Agree with Wiktor, can we add support for this at the module level?

Reply:
We had the discussion some time ago about this with @ludoo and the conclusion was, that as the resources are the same from GCP perspective, we don't need a separate module (that is, two cloud-run modules in Fabric), just one that uses new API.

So I think that time has come to do that.

I'm just not sure whether to keep the same name, especially if we manage to keep the same interface. Because then - it will just work, but if you happen to have instances in the state, it will be a mess. Usually backward incompatible changes in Fabric announce themselves not only in `CHANGELOG.md` but also as errors before apply.

Reply:
Good point Wiktor, what if we created a new module, and then deprecated the old one in the next release?

Reply:
Not what we discussed, but maybe it's time we became a little bit more conservative at times...

Reply:
This is what I was afraid of, releasing and deprecating may be not that smooth... So I wanted to move forward with this blueprint using the resource to not be blocked. Updating it once the new module is available would be straightforward.
It's your call, folks.